README.md

```diff
@@ -278,6 +284,10 @@ represents a sample library. Each read is counted towards all the annotated
 miRNA species it aligns to, with 1/n, where n is the number of genomic and/or
 transcriptomic loci that read aligns to.
 
+5. ASCII-style pileups of read alignments for the whole run, across libraries,
+and across subset of libraries - if specified - against the regions in the
+user-provided BED file, if any.
```

Comment:
"for the whole run, across libraries and across subsets of libraries" is ambiguous/vague. Maybe better something like this:

**OPTIONAL** ASCII-style pileups of read alignments produced for individual libraries, combinations of libraries and/or all libraries of a given run.  The exact number and nature of outputs depends on the workflow inputs/parameters.